main.css

```diff
@@ -1,17 +1,105 @@
+* {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+#book-list {
+  border: 3px solid darkgray;
+  border-radius: 5px;
+  width: 800px;
+  height: 250px;
+  overflow: auto;
+}
+
+.delete-book {
+  border-radius: 10px;
+  cursor: pointer;
+  border: 1px solid black;
+  width: 80px;
+  box-shadow: 2px 2px 2px black;
+}
+
+.delete-book:hover {
+  background-color: #ff5959;
+  color: white;
+}
+
+#add-book {
+  align-self: flex-end;
+  border-radius: 10px;
+  cursor: pointer;
+  border: 1px solid black;
+  width: 80px;
+  height: 35px;
+  box-shadow: 2px 2px 2px black;
+}
+
+#add-book:hover {
+  background-color: rgba(52, 63, 223, 0.986);
+  color: white;
+}
+
+.book-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  color: white;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  height: 35px;
+  padding: 8px;
+}
+
+.book:nth-child(odd) {
+  background-color: rgba(15, 26, 168, 0.979);
+}
+
+.book:nth-child(even) {
+  background-color: rgba(15, 25, 168, 0.705);
+}
+
 .body-center {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
 }
 
+.horizon {
+  height: 1px;
+  width: 400px;
+  background-color: darkgray;
+}
+
 .form {
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
+  align-items: center;
   justify-content: center;
+  max-width: 50%;
+  width: 15%;
 }
 
-.book {
-  margin: 1rem;
+.adjust-margin {
+  margin: 15px 0;
+  text-align: center;
+}
+
+input {
+  width: 100%;
+  font-size: 1.1em;
+  outline: none;
+  padding: 5px;
+  border-radius: 5px;
+  border: 1px solid darkgray;
+}
+
+input:focus-visible {
+  border: 2px solid rgba(52, 63, 223, 0.986);
+}
+
+.hidden {
+  display: none;
 }
```

Comment:
- [ ] Let us to try to get closer to the design provided.

-----------------------------------Yours--------------------------------

![storag2](https://user-images.githubusercontent.com/55635977/133453950-c9ed993d-15a9-447f-8e81-cf15e13090b9.png)
----------------------------------Original--------------------------------
![storag3](https://user-images.githubusercontent.com/55635977/133453971-6962c2db-2ca7-4368-a800-d30853615bcf.png)